tests/time_tests/test_runner/conftest.py

```diff
@@ -0,0 +1,81 @@
+"""
+Basic high-level plugin file for pytest.
+
+See [Writing plugins](https://docs.pytest.org/en/latest/writing_plugins.html)
+for more information.
+
+This plugin adds the following command-line options:
+
+* `--test_conf` - Path to test configuration file. Used to parametrize tests.
+  Format: YAML file.
+* `--exe` - Path to a timetest binary to execute.
+* `--niter` - Number of times to run executable.
+"""
+
+# pylint:disable=import-error
+import pytest
+from pathlib import Path
+import yaml
+
+
+# -------------------- CLI options --------------------
+
+def pytest_addoption(parser):
+    """Specify command-line options for all plugins"""
+    parser.addoption(
+        "--test_conf",
+        type=Path,
+        help="Path to test config",
+        default=Path(__file__).parent / "test_config.yml"
+    )
+    parser.addoption(
+        "--exe",
+        required=True,
+        dest="executable",
+        type=Path,
+        help="Path to a timetest binary to execute",
+    )
+    parser.addoption(
+        "--niter",
+        type=int,
+        help="Number of iterations to run executable and aggregate results",
+        default=3
+    )
+    # TODO: add support of --mo, --omz etc. required for OMZ support
+
+
+@pytest.fixture(scope="session")
+def test_conf(request):
+    """Fixture function for command-line option."""
+    return request.config.getoption('test_conf')
+
+
+@pytest.fixture(scope="session")
+def executable(request):
+    """Fixture function for command-line option."""
+    return request.config.getoption('executable')
+
+
+@pytest.fixture(scope="session")
+def niter(request):
```

Comment:
I can't find where it is used. Why do we need it?

Reply:
It's used here - https://github.com/openvinotoolkit/openvino/blob/7ad3a7feb2a6abf551349c50953dabddc3ad5302/tests/time_tests/test_runner/test_timetest.py#L36

Required for run_timetest